addons/website/static/src/less/website.ui.components.less

```diff
@@ -180,67 +208,206 @@ body .modal {
             }
         }
 
-        ~ .modal-backdrop {
+        + .modal-backdrop {
             opacity: 0.8;
         }
     }
 
+    // TOP MENU EDITOR
+    &.o_contentMenu_modal {
+        .oe_menu_editor {
+            ul {
+                padding-left: 37px;
+            }
+
+            li {
+                margin-top: -1px;
+
+                .input-group-addon {
+                    border-radius: 0;
+                }
+            }
+        }
+
+        // 'ADD MENU ITEM' MODAL BACKDROP
+        ~ .modal-backdrop {
+            opacity: 0.3;
+            z-index: @zindex-modal + 1;
+        }
```

Comment:
@qsm-odoo ,  So... the problem is that through the "Edit Menu" modal, the user can launch a second modal to edit the menu item link.
This will cause to have 2 modal active and, because of bootstrap z-index rules, the backdrops are at the same level.

The **visual result** is a "increased opacity" of the first backdrop while the two modals are both visible... it doesn't have so much sense to me.

This fix will move the second modal "over" the first one. Another solution would be to totally hide the second backdrop...